components/liquidity/ExitPoolForm.tsx

```diff
@@ -236,10 +234,10 @@ const ExitPoolForm = ({
                     } else if (value <= 0) {
                       return "Value cannot be zero or less";
                     } else if (
-                      poolStatus.toLowerCase() === "active" &&
+                      market.status.toLowerCase() !== "resolved" &&
                       poolAssetBalance.minus(value).lessThanOrEqualTo(0.01)
                     ) {
-                      return "Pool cannot be emptied completely whilst it's active";
+                      return "Pool cannot be emptied completely before market gets resolved";
```

Comment:
Maybe this is slightly better?
```suggestion
                      return "Pool cannot be emptied completely before the market resolves";
```